### DISCUSSION TITLE
以上cookie会导致too long吗

### DISCUSSION THREAD

**Discussion by GUANchenchen023** [fireboomio/product-manual#30] [Q&A]
![image](https://github.com/fireboomio/product-manual/assets/135700721/55591da4-bc2c-4848-bccb-ba2c2e9455f4)

**Comment by GUANchenchen023** [comment]
too long是claims中的消息太长导致的。参考下这个文档：https://ansons-organization.gitbook.io/product-manual/huan-jing-zhun-bei#zi-bu-shu-casdoor

**Reply by hsluoyz** [reply]
可以在Casdoor应用配置页面，选择JWT-Empty: https://casdoor.org/docs/token/overview , 可以显著减少claims大小
